sc2reader/objects.py

```diff
@@ -164,10 +164,13 @@ class Player(object):
     :param dict detail_data: The detail data associated with this player
     :param dict attribute_data: The attribute data associated with this player
     """
-    def __init__(self, pid, detail_data, attribute_data):
+    def __init__(self, pid, slot_data, detail_data, attribute_data):
```

Comment:
have to pass `slot_data` through to `Player` because that's where the commander name is